datafusion/core/tests/sqllogictests/src/setup.rs

```diff
@@ -212,3 +220,311 @@ fn register_nan_table(ctx: &SessionContext) {
     .unwrap();
     ctx.register_batch("test_float", data).unwrap();
 }
+
+pub async fn register_timestamps_table(ctx: &SessionContext) {
```

Comment:
I think this is code we can potentially port to sql (using `arrow-typeof`), but we can also handle that as a follow on PR too